tests/conftest.py

```diff
@@ -55,6 +45,111 @@ def pytest_ignore_collect(path, config):
             return True
 
 
+def pytest_configure(config):
+    config.addinivalue_line(
+        'markers',
+        'bdb(): use bigchaindb')
+
+
+@pytest.fixture(autouse=True)
+def _bdb_marker(request):
+    if request.keywords.get('bdb', None):
+        request.getfixturevalue('_bdb')
+
+
+@pytest.fixture(autouse=True)
+def _restore_config(_configure_bigchaindb):
+    from bigchaindb import config, config_utils
+    config_before_test = copy.deepcopy(config)
+    yield
+    config_utils.set_config(config_before_test)
+
+
+@pytest.fixture
+def _restore_dbs(request):
```

Comment:
The `rethinkdb` and `test` databases are default rethinkdb databases, not related with bigchaindb.

After the tests finish we should delete the `bigchaindb_test_*` databases. Also I don't remember if this was being done before but we should delete the `bigchaindb_test_*` tables when starting to run the tests. This accounts for the fact that we may have interrupted the tests `Ctrl-c` in a previous run.